src/verify.ts

```diff
@@ -36,7 +38,18 @@ export class Verifier {
   ): Promise<boolean> {
     signature = signature.trim();
 
-    return verifyBlob(certificate, payload, signature);
+    return crypto.verifyBlob(certificate, payload, signature);
+  }
+
+  public async verifyDSSE(bundle: SigstoreDSSEBundle): Promise<boolean> {
+    const payloadType = bundle.attestation.payloadType;
+    const payload = Buffer.from(bundle.attestation.payload, 'base64');
+    const paeBuffer = pae(payloadType, payload);
+
+    const signature = bundle.attestation.signatures[0].sig;
```

Comment:
Potential for an oob here. Does it matter? The error is being swallowed at https://github.com/sigstore/sigstore-js/pull/64/files#diff-2304ffbbb1cb4987906a47b08b7c3b590fc542f4247b733844049776fd8591f2R79-R84

Reply:
Or this doesn't matter in JS...

Reply:
Fixed